test/containerfiles/container-passes-privileged.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
Everything else looks good, but I'm not sure calling this this 'container-passes' is necessarily correct. Because it won't pass without the Pyxis call and removing RunsAsRoot, right?

Reply:
I guess its to test that RunAsRoot was skipped ?

Reply:
@bcrochet That's true, I wasn't really sure what to call it. I just wanted to show the container that I used to test that went along with the screen shot shared. Thoughts on naming it similar to your suggestion below `container-passes-with-root-exception`? Or is it better to not include this container in the codebase?

Reply:
I don't think we need to add this file at all, as you say. This case can be covered by the container-fails image.

Reply:
I ended up removing this, I don't think anyone else will use this container in the future, but was needed to show the tests I did.